FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta2/c_sdk/platform/freertos/mbedtls/mbedtls_freertos_port.c

```diff
@@ -142,6 +214,17 @@ int mbedtls_platform_mutex_unlock( mbedtls_threading_mutex_t * pMutex )
 
 /*-----------------------------------------------------------*/
 
+/**
+ * @brief Function to generate a random number.
+ *
+ * @param[in] data Callback context.
+ * @param[out] output The address of the buffer that receives the random number.
+ * @param[in] len Maximum size of the random number to be generated.
+ * @param[out] olen The size, in bytes, of the #output buffer.
```

Comment:
Should it be updated to this?
```suggestion
 * @param[out] olen The size, in bytes, of the generated random number in the #output buffer.
```

Reply:
Same for the `mbedtls_hardware_poll` function

Reply:
This was taken from the official documentation of mbedtls as is. I think the existing explanation hold good as well as the info about ouput buffer is available 2 lines above. Same for  mbedtls_hardware_poll